web/files/upload.go

```diff
@@ -0,0 +1,57 @@
+// Package files is for storing files on the cozy, including binary ones like
+// photos and movies. The range of possible operations with this endpoint goes
+// from simple ones, like uploading a file, to more complex ones, like renaming
+// a folder. It also ensure that an instance is not exceeding its quota, and
+// keeps a trash to recover files recently deleted.
+package files
+
+import (
+	"bytes"
+	"crypto/md5"
```

Comment:
It looks like travis build is red because gometalinter uses a checker called "gas" that verify some security rules, including that no DES, RC4 or MD5 is used. It's possible to disable those checks with a `// #nosec` comment, as explained here: https://github.com/HewlettPackard/gas#annotating-code